steering_functionsConfig.cmake

```diff
@@ -0,0 +1,7 @@
+include(CMakeFindDependencyMacro)
+
+# Same syntax as find_package
+find_dependency(Eigen3 REQUIRED)
+
+# Add the targets file
+include("${CMAKE_CURRENT_LIST_DIR}/steering_functionsTargets.cmake")
```

Comment:
Same here: snake case and camel case.

Reply:
Same as above, I guess this could be `steering_functions-targets.cmake` if that is better?